main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -129,7 +129,7 @@ object Weeder2 {
         val qname = Name.QName(tree.loc.sp1, namespace, ident, tree.loc.sp2)
         val res = Validation.success(UseOrImport.Use(qname, alias, tree.loc))
         // Check for illegal alias
-        if (ident.name.charAt(0).isUpper != alias.name.charAt(0).isUpper) {
+        if ((ident.name.nonEmpty && alias.name.nonEmpty) && ident.name.charAt(0).isUpper != alias.name.charAt(0).isUpper) {
```

Comment:
Do we need to check for this pattern elsewhere? Can you search for ident.name usages?

Reply:
I did such a search when I found that issue, with no results 👍 I think I searched both for `charAt` and also just `0` to see if I was indexing in a bad way anywhere.

Reply:
I did such a search when I found that issue, with no results 👍 I think I searched both for `charAt` and also just `0` to see if I was indexing in a bad way anywhere.